src/views/v2/components/events-bar/search/keyword.php

```diff
@@ -29,7 +29,23 @@ class="tribe-common-form-control-text__input tribe-events-c-search__input tribe-
 		id="tribe-events-events-bar-keyword"
 		name="tribe-events-views[tribe-bar-search]"
 		value="<?php echo esc_attr( tribe_events_template_var( [ 'bar', 'keyword' ], '' ) ); ?>"
-		placeholder="<?php esc_attr_e( 'Search for events', 'the-events-calendar' ); ?>"
-		aria-label="<?php printf( esc_attr__( 'Enter Keyword. Search for %s by Keyword.', 'the-events-calendar' ), tribe_get_event_label_plural() ); ?>"
+		placeholder="<?php
+			echo esc_attr(
+				sprintf(
+					/* translators: %s: Events (plural). */
+					__( 'Search for %s', 'the-events-calendar' ),
+					tribe_get_event_label_plural()
+				)
+			);
+		?>"
```

Comment:
* :no_entry_sign: Line indented incorrectly; expected 3 tabs, found 2
* :no_entry_sign: Closing PHP tag must be on a line by itself

<sub>hash: 100ac72eb2a8cbfac0397a32472ad0b1</sub>